lib/internal/streams/readable.js

```diff
@@ -1436,10 +1524,15 @@ ObjectDefineProperties(ReadableState.prototype, {
   paused: {
     __proto__: null,
     get() {
-      return this[kPaused] !== false;
+      return (this[kState] & kPaused) !== 0;
     },
     set(value) {
-      this[kPaused] = !!value;
+      this[kState] |= kHasPaused;
```

Comment:
why assume has paused to be true? if assigning it to false the first time?

Reply:
this basically means that it is not null.